src/main/java/com/quiz/FindPersonByUuidAndAskQuestion.java

```diff
@@ -0,0 +1,91 @@
+package com.quiz;
+
+import com.Utils.AppProperties;
+import com.enumex.EGender;
+import com.enumex.EStatus;
+import com.person.Person;
+import com.Utils.CommonUtils;
+import com.service.PersonService;
+
+import java.io.*;
+import java.util.Date;
+import java.util.Scanner;
+import java.util.UUID;
+
+public class FindPersonByUuidAndAskQuestion {
+
+    private static final String VALID_UUID = "Enter a valid uuid";
+    private static final String UUID_PATTERN = "^[0-9A-Fa-f]{8}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{4}-[0-9A-Fa-f]{12}$";
+    private static final String INVALID_UUID = "You have entered wrong or invalid uuid";
+    private static final String PERSON_UUID_BELONGS_TO = "This uuid belongs to %s\n";
+    public static final String ANSWER_NOW_QUESTION = "Answer now %s's question...\n";
+    public static final String LINE_SEPARATOR = "*******************";
+    public static final String RESULT = "Your result is %d";
+
+    private static PersonService personService = new PersonService();
+    private static Question question = new Question("test");
+    private static Scanner scanner = new Scanner(System.in);
+
+    public static void readFromFile() {
```

Comment:
1.What do you read from file? Method naming is poor
2.This method does 2 things - 1.Populates some list 2.Asks some questions. Methods should have only one responsibility. So you can extract one of the two things in another method
3.Don't you think that populating the list of persons is a responsibility of the PersonService? I mean that this method should probably not be here
4.Why static?